php/php.editor/tools/ASTPHP5Parser.cup

```diff
@@ -382,6 +409,7 @@ terminal T_POW_EQUAL;
 terminal T_ELLIPSIS;
 terminal T_COALESCE;
 terminal T_COALESCE_EQUAL;
+terminal T_ATTRIBUTE;
```

Comment:
To avoid the incompatible api change, always add new one to the last position.

Reply:
I see.